org.eclipse.lsp4e/plugin.xml

```diff
@@ -195,6 +202,10 @@
                   definitionId="org.eclipse.lsp4e.textSelectionHasLanguageServer">
             </reference>
          </activeWhen>
+      </handler>
+      <handler
```

Comment:
This hanndler should be enabled only when current selection or editor can be assisted by language server.